migrations/20200409232153_rewrite_twitter_urls.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class RewriteTwitterUrls extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-abstractmigration-class
+     *
+     * The following commands can be used in this method and Phinx will
+     * automatically reverse them when rolling back:
+     *
+     *    createTable
+     *    renameTable
+     *    addColumn
+     *    renameColumn
+     *    addIndex
+     *    addForeignKey
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function change()
+    {
+        $sql = "UPDATE messages 
```

Comment:
my big fear with this change is that it'll be pretty heavy in some deployments... like my ridiculous testing one with like 300k tweeets :/